### PR TITLE
Setting up the repo

### PR DESCRIPTION
**Pull request by sithankanna**
To test this PR follow the setup instructions in the README.

After the setup, running

```bash
poetry run main
```
should result in

```bash
[2021-08-06 XX:XX:XX][crypto.console][INFO] - Starting experiments.exp_hello_world
[2021-08-06 XX:XX:XX][experiments.exp_hello_world.experiment][DEBUG] - Hello World
[2021-08-06 XX:XX:XX][experiments.exp_hello_world.experiment][DEBUG] - ['aax', 'aofex', 'ascendex', 'bequant', 'bibox', 'bigone', 'binance', 'binancecoinm', 'binanceus', 'binanceusdm', 'bit2c', 'bitbank', 'bitbay', 'bitbns', 'bitcoincom', 'bitfinex', 'bitfinex2', 'bitflyer', 'bitforex', 'bitget', 'bithumb', 'bitmart', 'bitmex', 'bitpanda', 'bitso', 'bitstamp', 'bitstamp1', 'bittrex', 'bitvavo', 'bitz', 'bl3p', 'braziliex', 'btcalpha', 'btcbox', 'btcmarkets', 'btctradeua', 'btcturk', 'buda', 'bw', 'bybit', 'cdax', 'cex', 'coinbase', 'coinbaseprime', 'coinbasepro', 'coincheck', 'coinegg', 'coinex', 'coinfalcon', 'coinfloor', 'coinmarketcap', 'coinmate', 'coinone', 'coinspot', 'crex24', 'currencycom', 'delta', 'deribit', 'digifinex', 'eqonex', 'equos', 'exmo', 'exx', 'flowbtc', 'ftx', 'gateio', 'gemini', 'gopax', 'hbtc', 'hitbtc', 'hollaex', 'huobi', 'huobijp', 'huobipro', 'idex', 'independentreserve', 'indodax', 'itbit', 'kraken', 'kucoin', 'kuna', 'latoken', 'lbank', 'liquid', 'luno', 'lykke', 'mercado', 'mixcoins', 'ndax', 'novadax', 'oceanex', 'okcoin', 'okex', 'okex3', 'okex5', 'paymium', 'phemex', 'poloniex', 'probit', 'qtrade', 'ripio', 'stex', 'therock', 'tidebit', 'tidex', 'timex', 'upbit', 'vcc', 'wavesexchange', 'whitebit', 'xena', 'yobit', 'zaif', 'zb']
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,135 @@
 # crypto
+
+## First Time Setup
+
+1. Install PyEnv to manage python installations. `brew install pyenv`
```
Comment (davesnowdon) [inline README.md:5]: Maybe also mention `curl https://pyenv.run | bash` for non-Mac users or point to https://github.com/pyenv/pyenv-installer

File: README.md [hunk 2]
```diff
@@ -1 +1,135 @@
...
+
+5. Check if your system python is the same `python --version`
+
+6. Install poetry
+
+ 
+
+    ```bash
+
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
```
Comment (davesnowdon) [inline README.md:31]: According to the docs get-poetry.py will soon be deprecated and if we want to install newer versions of poetry we should use install-poetry.py instead as described here https://python-poetry.org/docs/master/#installation

**Review by davesnowdon — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (sithankanna) [reply README.md]: Fixed!

File: README.md (hunk 1 above)
Comment (sithankanna) [reply README.md]: Fixed! Thanks!

**Review by davesnowdon — APPROVED** [review]

**Pull request merged**